artifacts/otg.proto

```diff
@@ -12078,6 +12103,9 @@ service Openapi {
   rpc SendPing(SendPingRequest) returns (SendPingResponse) {
     option (rpc_meta).description = "API to send an IPv4 and/or IPv6 ICMP Echo Request(s) between endpoints. For each endpoint 1 ping packet will be sent and API shall wait for ping response to either be successful or timeout. The API wait timeout for each request is 300ms.";
   }
+  rpc SetProtocolState(SetProtocolStateRequest) returns (SetProtocolStateResponse) {
+    option (rpc_meta).description = "Sets all configured protocols to intended state";
```

Comment:
started or stopped ?

Reply:
changed to mention `start` or `stop` state.